modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -3036,6 +3093,19 @@ private void recalculateAndImproveEstimatedSize(int improve) {
             }
         }
         this.estimatedSize = cnt == 0 ? 1d: tot * itemCount / cnt;
+        estSize = estimatedSize / itemCount;
+
+        if (keepRatio ) {
```

Comment:
Minor: there is an extra space before the `)`

Reply:
done